internal/cli/cmd/cluster/create.go

```diff
@@ -1335,6 +1368,49 @@ func (o *CreateOptions) buildAnnotation(cls *appsv1alpha1.Cluster) {
 	}
 }
 
+func (o *CreateOptions) buildBackupConfig(cls *appsv1alpha1.Cluster) error {
+	// set default backup config
+	o.BackupConfig = &appsv1alpha1.ClusterBackup{
+		Method: dataprotectionv1alpha1.BackupMethodSnapshot,
```

Comment:
User may not be aware of this default value, should we set a more noticeable default value for the backup-method flag?

Reply:
yes,  maybe set this default value in flags is ok. Before I want to know  which flag be visited, so I set them emtpy and now it' unncessary.